picker_wrapper_test.go

```diff
@@ -32,7 +32,9 @@ import (
 	"google.golang.org/grpc/status"
 )
 
-const goroutineCount = 5
+const (
+	goroutineCount = 5
+)
```

Comment:
This fits on one line, so you should revert this change.

Reply:
Sure, fixed.